backend/src/main/kotlin/no/uib/echo/DatabaseHandler.kt

```diff
@@ -61,7 +67,8 @@ class DatabaseHandler(private val dev: Boolean, private val testMigration: Boole
                         Registration,
                         Answer,
                         SpotRange,
-                        User
+                        User,
```

Comment:
Det må lages et migrasjonsskript i `src/resources/db.migration/` som starter med `V16__` (se de andre navnene i mappa). Der må du oprette tabellen `feedback`, se `V14__User.sql` for inspo. Dette skjer automatisk når man kjører lokalt med en fersk database, men ikke når man prøver å koble til en eksisterende database.